usaspending_api/references/migrations/0067_alter_office_office_code.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.0.4 on 2024-04-15 19:24
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("references", "0066_alter_office_unique_index_code_only"),
+    ]
+
+    operations = [
```

Comment:
We do already have a migration that accomplishes this, but it is done with a SQL script, so I'm not sure if it is registering in the Django state. Can you check if this causes any problems when applying migrations?